x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/components/datastream_section.tsx

```diff
@@ -45,23 +45,19 @@ export const OverviewDatastreamSection: React.FC = () => {
 
   return (
     <EuiFlexItem component="section">
-      <OverviewPanel>
-        <header>
-          <EuiTitle size="xs">
-            <h2>
-              <FormattedMessage
-                id="xpack.ingestManager.overviewPageDataStreamsPanelTitle"
-                defaultMessage="Datasets"
-              />
-            </h2>
-          </EuiTitle>
-          <EuiButtonEmpty size="xs" flush="right" href={getHref('data_streams')}>
-            <FormattedMessage
-              id="xpack.ingestManager.overviewPageDataStreamsPanelAction"
-              defaultMessage="View datasets"
-            />
-          </EuiButtonEmpty>
-        </header>
+      <OverviewPanel
+        title={i18n.translate('xpack.ingestManager.overviewPageDataStreamsPanelTitle', {
+          defaultMessage: 'Datasets',
+        })}
+        tooltip={i18n.translate('xpack.ingestManager.overviewPageDataStreamsPanelTooltip', {
+          defaultMessage:
+            'Data that your agents collect are organized into various datasets. You can access links to view this data in other applications.',
```

Comment:
TBH, I'm not sure what this means: "You can access links to view this data in other applications."